spaceship/lib/spaceship/connect_api/tunes/tunes.rb

```diff
@@ -1159,6 +1159,134 @@ def get_territories(filter: {}, includes: nil, limit: nil, sort: nil)
           params = tunes_request_client.build_params(filter: nil, includes: nil, limit: nil, sort: nil)
           tunes_request_client.get("territories", params)
         end
+
+        #
+        # appClips
+        #
+
+        def get_app_clips(app_id:, filter: {}, includes: nil, limit: nil, sort: nil)
+          params = tunes_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          tunes_request_client.get("apps/#{app_id}/appClips", params)
+        end
+
+        #
+        # appClipVersions
+        #
+
+        def get_app_clip_versions(app_clip_id:, filter: {}, includes: nil, limit: nil, sort: nil)
+          params = tunes_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          tunes_request_client.get("appClips/#{app_clip_id}/appClipVersions", params)
+        end
+
+        def get_app_clip_version_release_with_app_store_version(app_clip_version_id:, filter: {}, includes: nil, limit: nil, sort: nil)
+          params = tunes_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          tunes_request_client.get("appClipVersions/#{app_clip_version_id}/relationships/releaseWithAppStoreVersion", params)
+        end
+
+        def get_app_clip_app_store_review_detail(app_clip_version_id:, filter: {}, includes: nil, limit: nil, sort: nil)
+          params = tunes_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          tunes_request_client.get("appClipVersions/#{app_clip_version_id}/relationships/appClipAppStoreReviewDetail", params)
+        end
```

Comment:
There are those resources related to `appClipVersions` but didn't add models for them since I have no idea how to use them.